components/Order/index.rst

```diff
@@ -4,10 +4,12 @@ Order
 E-Commerce PHP library for creating and managing sales orders.
 
 .. toctree::
-   :numbered:
+   :maxdepth: 2
 
    installation
-   order_and_items
-   adjustments
+   models
+   interfaces
    order_states
-   summary
+   order_events
+   order_transitions
+   basic_usage
```

Comment:
I am thinking that maybe ``basic_usage`` should be before other chapters? This is more interesting to read after installation and then other chapters are "supplements" to this basic usage, for better understand. Thoughts?